### ISSUE TITLE
NTR: Positive glucose or lactulose hydrogen breath test (Triggers or responses: non-drug)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9549]
New term request Positive glucose or lactulose hydrogen breath test (nadea3_200428184201):   UMLS: 3495742

**Comment by pnrobinson** [comment]
@MickeySegal
are we talking about this
https://pubmed.ncbi.nlm.nih.gov/26992249/
and it is the case that one can use glucose or lactulose to the same end?

**Comment by MickeySegal** [comment]
That is the correct test.  This description indicates that they are something of package: https://www.brighamandwomens.org/medicine/gastroenterology-hepatology-and-endoscopy/endoscopy-procedures/hydrogen-breath-tests-for-lactose-intolerance-and-bacterial-overgrowth

**Comment by pnrobinson** [comment]
I will make the primary label 
Positive lactulose hydrogen breath test
with the synonyms
Positive glucose or lactulose hydrogen breath test
and
Positive glucose hydrogen breath test

**Comment by pnrobinson** [comment]
Actually, I think it may be sufficient to make them synonyms of the existing term
[Small intestinal bacterial overgrowth HP:0033163](https://hpo.jax.org/app/browse/term/HP:0033163)

The HPO usually does not distinguihs between different ways of measuring the same thing and AFAIK this test only assays SI bacterial overgrowth.

See also When a patient presents with signs and symptoms concerning SIBO, the diagnosis garners further support with a positive carbohydrate breath test or a bacterial concentration of more than 1000 colony-forming units/mL in a jejunal aspirate culture (https://www.ncbi.nlm.nih.gov/books/NBK546634/)

Thoughts?

**Comment by MickeySegal** [comment]
This is abnormal in other conditions such as lactose intolerance and celiac disease.  So the SIBO designation doesn't cover the richness well.

Also, if one were doing Natural Language Processing and the chart only mentioned the hydrogen breath test, NLP wouldn't pick this up looking for SIBO.  Hydrogen production is a test abnormality and the SIBO is our assessment of its bacterial cause.  

BTW, I see that we have another finding brumme_220429155500 "Positive lactose hydrogen breath test" that based on the Brigham link above is clearly the same thing, so I merged it into nadea3_200428184201.

**Comment by pnrobinson** [comment]
@MickeySegal  I thought that the lactose breath test was for lactose intolerance? Probably these two concepts need to be separate?

**Comment by MickeySegal** [comment]
The Brigham link that I gave suggests that the tests are done for different purposes but not very different.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000259
- label: Positive hydrogen breath test
- parents: [Functional abnormality of the gastrointestinal tract (HP:0012719)](https://hpo.jax.org/app/browse/term/HP:0012719)
- pmid: PMID:21860825
- def: Abnormally elevated quantiy of hydrogen measured by a hydrogen breath test (HBT). HBTs administer substrates such as glucose, lactulose, lactose and fructose and subsequently measure hydrogen exhaled in the breath using a gas chromatograph.
- comment: Bacteria, especially anaerobic, colonizing the large bowel in health and small bowel in diseased conditions produce hydrogen by fermentation of unabsorbed carbohydrates. Though small amount of hydrogen is produced from limited amounts of unabsorbed carbohydrate reaching the colon, large amounts of hydrogen may be produced if there is malaborption of carbohydrate in the small intestine allowing larger amount to reach the colon or if there is excess of bacteria in the small bowel. The hydrogen produced by the bacteria is absorbed through the wall of the small or large intestine or both. The hydrogen-containing blood travels to the lungs where the hydrogen is released and exhaled in the breath where it can be measured.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9549
- synonyms: Positive glucose hydrogen breath test(NONE)|Positive glucose or lactulose hydrogen breath test(NONE)|Positive lactulose hydrogen breath test(NONE)

**Comment by MickeySegal** [comment]
I see that there is also https://hpo.jax.org/app/browse/term/HP:4000100, which seems a subset of HP:6000259
